web/app/scripts/src/services/api.es.js

```diff
@@ -33,6 +33,12 @@ module.service(moduleProperties.componentName, [
             getForm: function (url) {
                 return $http.get(url)
                     .then((response) => response.data);
+            },
+            post: function (baseUrl, data) {
+                const endpoint = 'respond';
```

Comment:
We have `const endpoint = 'forms';` just above. Seems a bit pointless to do this differently.